### ISSUE TITLE
imextract_init fails for CV7000 data with multiple "timelines"

### ISSUE THREAD

**Issue by scottberry** [pelkmanslab/TissueMAPS-OLD#2]
imextract_init fails when preprocessing 378 tifs from CellVoyager. See screenshot for details.
Image details: 3 channels, single image tifs from 96-well plate 9 sites per well.
Data is available for testing at 172.23.67.79/data/Data/Users/Scott/20170106-EU-CT-opt-Scott
I also tried to resubmit this data as a 384-well plate, however the result was the same.

![screen shot 2017-01-11 at 14 33 53](https://cloud.githubusercontent.com/assets/5407848/21855221/f4642058-d81d-11e6-9756-7a133ae71a02.png)

**Comment by scottberry** [comment]
This may be related to having multiple "timelines" in the Yokogawa acquisition for this plate. i.e. some wells in the plate have different numbers of acquisition channels. I successfully uploaded and preprocessed/created pyramids for two wells of this plate which have the same number of aquisition channels, without providing the metadata files. This is however broken (at metaconfig) if I provide the metadata files.

**Comment by hackermd** [comment]
What's the name of the experiment? I will have a look.

**Comment by scottberry** [comment]
Image details: 3 channels, single image tifs from 96-well plate 9 sites per well.
Data is available for testing at 172.23.67.79/data/Data/Users/Scott/20170106-EU-CT-opt-Scott
I also tried to resubmit this data as a 384-well plate, however the result was the same.

**Comment by hackermd** [comment]
Can you please add one of my public keys to `~/.ssh/authorized_keys` to provide me access to your machine. You can get the keys from Github:

```
wget -O - https://api.github.com/users/hackermd/keys
```

**Comment by scottberry** [comment]
Which machine? My local machine? Why do you need this?

The data is all on sciencecloud storage and I am doing uploads from instances launched from sciencecloud instance snapshot tm_client

Is there some issue with permissions on the data? I could change this on science-cloud storage, if required?

Scott

**Comment by hackermd** [comment]
I was referring to machine `172.23.67.79`. Ok, got it - its the new storage server, so I can mount it..

**Comment by scottberry** [comment]
Yes, it would be nice to give the new storage a name.

**Comment by hackermd** [comment]
@alecrimi I agree it would be nice to give it a domain name similar to the already existing sciencecloudstorage.pelkmanslab.org/pelkmanslab-share-1`

**Comment by scottberry** [comment]
@alecrimi perhaps best to wait until the storage is partitioned (next week?) and give a nice new series of names to the different partitions

**Comment by hackermd** [comment]
Some inspiration: [cities of Games of Thrones](http://gameofthrones.net/world-of-westeros/37-cities.html) :)

**Comment by scottberry** [comment]
@alecrimi, very good.
